components/AboutMe.js

```diff
@@ -95,14 +95,33 @@ export default function AboutMe() {
           </div>
           {/* Text area */}
           <div className="col-span-1 md:col-span-2">
-            {userData.about.description?.map((desc, idx) => (
-              <p
-                key={idx}
-                className="text-xl text-gray-700 mb-4 dark:text-gray-300 "
+            <p className="text-xl text-gray-700 mb-4 dark:text-gray-300">
+              👋🏼 Hi there! I'm Turner. Welcome to my site 🙂
+            </p>
+            <p className="text-xl text-gray-700 mb-4 dark:text-gray-300">
+              I'm a professional software engineer with two years' full-stack
+              experience building web and mobile apps. I'm especially inspired
+              by the positive impact possible at the intersection of tech,
+              government, and civic engagement, but can find joy in any cause
+              using software to help improve lives.
+            </p>
+            <p className="text-xl text-gray-700 mb-4 dark:text-gray-300">
+              When I'm not coding, you can find me at the gym, curled up with a
+              good (or sometimes bad) sci-fi novel, or crafting new adventures
+              for my D&D campaigns ⚔️🤓
+            </p>
+            <p className="text-xl text-gray-700 mb-4 dark:text-gray-300">
+              🙋‍♂️ <strong>I'm looking for a job!</strong> I thrive in
+              environments with humble, curious, kind folks. If you think I
+              might be a fit, please{" "}
+              <a
+                href="mailto:jturnerallen@gmail.com"
+                className="text-gray-800 border-b-2 border-gray-800 dark:border-gray-300 font-bold dark:text-gray-300"
```

Comment:
Small nit: wdyta throwing a hover style on this? Super small, but a nice micro-animation can make your site "pop" subtly :)

For example: `hover:border-indigo-500 hover:text-indigo-500 transition-all`

(the `transition-all` just adds a slight transition animation from/to the hover color)

Reply:
Added hover style for all three similar links on this page, also added image hover text for the tech stack icons (will revisit this with a higher quality system later)